test/src/specs/relay/transaction_relay_low_fee_rate.rs

```diff
@@ -1,95 +1,47 @@
 use crate::node::{connect_all, exit_ibd_mode};
-use crate::util::check::is_transaction_committed;
+use crate::util::cell::{as_input, as_output, gen_spendable};
 use crate::util::log_monitor::monitor_log_until_expected_show;
-use crate::utils::wait_until;
-use crate::{Node, Spec, DEFAULT_TX_PROPOSAL_WINDOW};
+use crate::{Node, Spec};
 use ckb_fee_estimator::FeeRate;
-use ckb_types::{core::TransactionView, packed, prelude::*};
-use log::info;
-use std::fs;
+use ckb_types::core::TransactionBuilder;
 
 pub struct TransactionRelayLowFeeRate;
 
 impl Spec for TransactionRelayLowFeeRate {
-    crate::setup!(num_nodes: 3);
+    crate::setup!(num_nodes: 2);
 
     fn run(&self, nodes: &mut Vec<Node>) {
         exit_ibd_mode(nodes);
         connect_all(nodes);
 
         let node0 = &nodes[0];
         let node1 = &nodes[1];
-        let node2 = &nodes[2];
 
-        info!("Generate new transaction on node1");
-        node1.generate_blocks((DEFAULT_TX_PROPOSAL_WINDOW.1 + 2) as usize);
-        let tx = node1.new_transaction_spend_tip_cellbase();
-        node1.submit_transaction(&tx);
-        let hash = tx.hash();
-        // confirm tx
-        node1.generate_blocks(20);
-        let ret = wait_until(10, || is_transaction_committed(node1, &tx));
-        assert!(ret, "send tx should success");
-        let tx: TransactionView = packed::Transaction::from(
-            node1
-                .rpc_client()
-                .get_transaction(hash.clone())
-                .unwrap()
-                .transaction
-                .inner,
-        )
-        .into_view();
-        let capacity = tx.outputs_capacity().unwrap();
-
-        info!("Generate zero fee rate tx");
-        let tx_low_fee = node1.new_transaction(hash);
-        // Set to zero fee
-        let output = tx_low_fee
-            .outputs()
-            .get(0)
-            .unwrap()
-            .as_builder()
-            .capacity(capacity.pack())
-            .build();
-        let tx_low_fee = tx_low_fee
-            .data()
-            .as_advanced_builder()
-            .set_outputs(vec![])
-            .output(output)
+        let cells = gen_spendable(node0, 1);
+        // As for `low_fee`, which is `inputs.total_capacity == outputs.total_capacity`,
```

Comment:
`inputs.total_capacity == outputs.total_capacity` more like zero_fee, but of course it also make sense in this case.